design/FY2019/NFP-DOAS_MultiAHUs.md

```diff
@@ -0,0 +1,513 @@
+# **Model DOAS Supplying Air to Inlets of Multiple AHUs**
+
+## Lixing Gu
+## Floirda Solar Energy Center
+
+### First draft: 3/11/19
+
+### **Justification for Feature Update:**
+
+This new feature will allow EnergyPlus to model an outside air system supplying ventilation air to multiple air handlers.
+
+Currently a DOAS (dedicated outdoor air system) in EnergyPlus is a normal AirLoopHVAC (aka air loop)  object with the recirculated air flow set to zero. In EnergyPlus the air loops are independent of each other: one air loop can not provide air to another. Outside air is provided to each air system by the AirLoopHVAC:OutdoorAirSystem object. Each outdoor air system can supply only one air loop. All this means that EnergyPlus can't simulate a common HVAC layout: a single multi-floor DOAS serving, through an air shaft, multiple single floor air handling units.
+
+FSEC will implement the new feature based on request shown below:
+
+#### Modal DOAS to Multiple Air Handling Units
+
+o	This feature develops the modeling and simulation approach for a dedicated outdoor air system (DOAS) connected to multiple air handling units (AHUs). Many buildings have a separate DOAS system that feeds outdoor air directly to individual AHUs on each building floor. Currently EnergyPlus can only model a DOAS delivering outdoor air directly to zones or to the inlet or outlet of zone equipment acting as terminal units. This feature will allow a single DOAS to supply air to the outdoor air inlet of multiple air systems.
+
+o	Ability to attach one DOAS to multiple AirLoopHVAC objects would be helpful to model: DOAS connected to multiple rooftop units (or) multiple SZVAV/SZCV units
+
+o	This feature will develop the modeling and simulation approach for a dedicated outdoor air system (DOAS) connected to multiple air handling units (AHUs). Many buildings have a separate DOAS system that feeds outdoor air directly to individual AHUs on each building floor. Currently EnergyPlus can only model a DOAS delivering outdoor air directly to zones or to the inlet or outlet of some zone equipment acting as terminal units. This feature will allow a single DOAS to supply air to the outdoor air inlet of multiple air systems.
+
+
+### **Conference Call Conclusions**
+None
+
+### **Overview**
+
+We propose 3 new objects and a revised object to accomplish the goal. 
+
+####New objects
+
+The proposed new objects are AirLoopHVAC:DedicatedOutdoorAirSystem, AirLoopHVAC:DedicatedOutdoorAirSystem:Mixer, and AirLoopHVAC:DedicatedOutdoorAirSystem:Splitter.
```

Comment:
Zone equipment already allow DOAS connected to inlets. It would be better for GUIs if the same object was used to connect to air loop equipment. The object is AirTerminal:SingleDuct:Mixer/Splitter. There are existing functions that calculate the mixing and other functions used for component sizing. The functions are in SingleDuct.cc and at the bottom of ReportSizingManager. The existing functions in SingleDuct.cc could be used by all equipment models. Also, we already have AirloopHVAC that can act as DOAS and don't need another dedicated object.

Reply:
@rraustad Thanks for your comment with quick feedback. The proposed new feature will have a central DOAS to serve multiple AirLoops. The name of AirTerminal:* may not be appropriated. I think any names with AirLoop are better. That is what is proposed. In addition, the proposed feature is a high level management to provide enough inputs to map required configurations. The detailed calculations will use existing functions. For example, any OA calculations will call existing OA controllers. 

The proposed object is similar with the AirLoop object with two main differences. The first difference is that the proposed object requires two nodes: inlet and outlet only, while the AirLoopHVAC object requires two pairs of node based supply and demand sides. In addition, the proposed object requires more inputs, such as setpoint manager, AirLoopHVAC:DedicatedOutdoorAirSystem:Mixer, and AirLoopHVAC:DedicatedOutdoorAirSystem:Splitter.

The second difference is that AirLoopHVAC is used to serve multiple zones, while the proposed object is used to serve multiple AirLoops. Since difference purposes are specified, it should use different names to specify how to use these objects properly. In addition, we have issues with zone mass balance in an AirLoopHVAC. The proposed object deals with multiple AirLoops only and there is no potential mass balance issue.  That is why I propose a new object name for a central DOAS system.